### PR TITLE
Integrate React

### PR DESCRIPTION
**Pull request by gauravchl**
Issue cover: https://github.com/codebuddies/cb-connect/issues/9

Using React and Blaze together doesn't make any sense. This PR is replacing the Blaze with React. 
Also there is a modular library for bootstrap in react world  https://react-bootstrap.netlify.com/getting-started/introduction/ using it instead legacy bootstrap in favor of react and modularity.

Currently app is ported to react and all blaze templates have been replaced by react components. [react-bootstrap](https://react-bootstrap.netlify.com/getting-started/introduction/) integration is remaining. Suggestions, helps are welcome :)

TODO:

- [x] Add React, remove Blaze
- [x] Add React router, remove flow router
- [x] Convert all blaze templates to react component
- [x] Install react-bootstrap.
- [x] Update FAQ component with react-bootstrap
- [x] (Gaurav Working)Update Apply component with react-bootstrap
- [x] (Gaurav working)Update Login component with react-bootstrap 
- [x] (Linda working) Update Landing component with react-bootstrap 
- [x] (Linda working) Update Navbar component with react-bootstrap 
- [x] Finally remove jquery, remove bootstrap scss `client/styles/bootstrap`

### CODE REVIEW

**Comment by lpatmo** [comment]
@gauravchl In order for me to push to your forked branch (wip-mvp), do you know if you'll need to give me permission on https://github.com/gauravchl/cb-connect? (Don't entirely remember how to push to a fork)

**Comment by gauravchl** [comment]
@lpatmo Try again please, i've added you as a collaborator.

**Comment by distalx** [comment]
Sorry, accidentally closed this PR when I remove the wip-mvp branch.

File: imports/ui/components/navbar.jsx [hunk 1]
```diff
@@ -10,18 +9,24 @@ class NavbarWrapper extends React.Component {
...
-          <Navbar.Toggle aria-controls="basic-navbar-nav"/>
-            <Nav className="mr-auto"></Nav>
-            <Nav.Link className="mr-sm-2" href="/faq">FAQ</Nav.Link>
-            <Button variant="outline-success" className="btn btn-primary btn-sm btn-border">Login</Button>
-          </Navbar>
-          {this.props.children}
-      </React.Fragment>
+      <Container>
+        <Navbar>
+          <Link to="/" className='navbar-brand'>
```
Comment (gauravchl) [inline imports/ui/components/navbar.jsx]: @lpatmo Made some tweaks. Using anchor tag directly for routing causes the page to reload.

**Comment by distalx** [comment]
@gauravchl , @lpatmo Thanks a lot for putting this all together 🚀 .

**Pull request merged**